README.md

```diff
@@ -143,47 +143,4 @@ is converted to
 ### Read CloudEvent Custom Format **Data** as a **byte[]**
```

Comment:
Unrelated to this change but saw it when reviewing this PR:

- Can we use a PS uuid generator function instead of giving an arbitrary `-id 'val'`  in the example?
- Can we provide an example of sending `httpStructured` event instead of just binary mode? We show both examples but don't provide an example of using the structured event and sending via http

Reply:
Created issue #18 for the proposed updates. I'll address in a separate PR.